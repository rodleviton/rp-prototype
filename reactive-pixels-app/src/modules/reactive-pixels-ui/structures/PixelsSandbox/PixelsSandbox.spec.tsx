import * as React from "react";
import { render } from "react-testing-library";
import WithRouter from "../../utils/test/WithRouter";
import WithTheme from "../../utils/test/WithTheme";
import { PixelsSandbox } from "./";

describe("<PixelsSandbox />", () => {
  it("should render <PixelsSandbox /> component", () => {
    const { container } = render(
      <WithRouter>
        <WithTheme>
          <PixelsSandbox src="" />
        </WithTheme>
      </WithRouter>
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
