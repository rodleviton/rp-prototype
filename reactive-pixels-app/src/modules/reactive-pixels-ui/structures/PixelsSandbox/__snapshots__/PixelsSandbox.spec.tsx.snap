// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PixelsSandbox /> should render <PixelsSandbox /> component 1`] = `
<div
  class="css-1efvhw7"
>
  <div
    class="css-1x10l6g css-1xn3edc"
  >
    <div
      class="css-1lz8w4n"
    >
      <iframe
        class="css-1vm9b5l css-q7lffx css-lnyvlk"
        frameborder="0"
        height="460"
        scrolling="no"
        src=""
        width="340"
      />
      <div
        class="css-nnvb6c css-k3oqj5"
      >
        <svg
          height="38"
          viewBox="0 0 460 516"
          width="38"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            fill="#9C9CA0"
          >
            <path
              class="css-1jnxzie"
              d="M333.372031,460 L447.102888,396.049894 C454.225767,391.973054 460,382.065204 460,373.932338 L460,249.593886 L356.224274,190 L355.657897,324.823726 C355.641368,328.729192 352.857585,333.480849 349.437,335.43865 L232,402.227074 L333.372031,460 Z"
            />
            <path
              class="css-1jnxzie"
              d="M210.509531,3.05293948 C217.583554,-1.02216524 233.04954,-1.01630663 240.11342,3.06249801 L447.204762,124.330903 C454.271377,128.411286 460,138.346818 460,146.527545 L460,246 L233.47922,115.241236 C230.076573,113.302921 224.591894,113.354919 221.221029,115.362032 L105,184.563481 L105,66.1376678 L210.509531,3.05293948 Z"
            />
            <path
              class="css-1jnxzie"
              d="M1,374.214594 C1,382.34674 5.78727134,392.236312 12.9351163,396.308633 L217.689432,512.962717 C224.8333,517.032772 236.399981,517.009813 243.528593,512.908989 L331,462.589963 L108.177655,334.741092 C104.750877,332.783693 101.972924,328.034824 101.972924,324.124429 L101.972924,68 L12.9872928,118.288501 C5.81460899,122.306552 0,132.156449 0,140.28838 L1,374.214594 Z"
            />
          </g>
        </svg>
      </div>
    </div>
    <div
      class="css-pvvspc"
    />
  </div>
  <button
    class="css-xxqrgn css-1t2o5q3"
  >
    <svg
      height="16px"
      viewBox="0 0 16 16"
      width="16px"
    >
      <g
        fill="none"
        fill-rule="evenodd"
        stroke="none"
        stroke-width="1"
      >
        <path
          d="M14,8.39687471 L8,15.0911045 L2,8.36928989 C1.2825,7.43701623 1,6.80724059 1,5.94509785 C1,4.3372297 2.2275,2.89019511 4,2.87663646 C5.458,2.86541552 7.111,4.25774799 8,5.30737405 C8.8655,4.29468361 10.542,2.87663646 12,2.87663646 C13.7255,2.87663646 15,4.3372297 15,5.94509785 C15,6.80724059 14.7765,7.48377017 14,8.39687471 L14,8.39687471 Z M12,2 C10.3335,2 9.0725,2.91450716 8,3.87015779 C6.9785,2.85606473 5.6665,2 4,2 C1.653,2 0,3.88698921 0,5.94509785 C0,7.04895849 0.4835,7.84377555 1.013,8.59090358 L7.2295,15.5600466 C7.927,16.2506023 8.059,16.2506023 8.7565,15.5600466 L14.987,8.59090358 C15.61,7.84377555 16,7.04895849 16,5.94509785 C16,3.88698921 14.347,2 12,2 L12,2 Z"
          fill="#FFFFFF"
        />
      </g>
    </svg>
  </button>
</div>
`;
